SPECS/dracut/50-noxattr.conf

```diff
@@ -0,0 +1,2 @@
+# disable xattr
+DRACUT_NO_XATTR=1
```

Comment:
Newline (please also update your editor to automatically include newlines in EOF)

Reply:
Done!